qiskit/util.py

```diff
@@ -95,3 +95,19 @@ def _has_connection(hostname, port):
         return True
     except Exception:  # pylint: disable=broad-except
         return False
+
+
+def _to_tuple(values):
+    """
+    Return the input, sorted, and as a tuple.
+
+    Args:
+        values (Union[int, Iterable[int]]): An integer, a list of ints, or a tuple of ints.
```

Comment:
depending on the type of `value`, right?

Reply:
Do you mean `values`? Yes, the allowed types of `values` are `Union[int, Iterable[int]]`

Reply:
Oh.. sorry. Forget it.